### PR TITLE
Devise-ify Players

### PR DESCRIPTION
**Pull request by jarednorman**
Okay. Sure, this might be ready... the tests pass.

### CODE REVIEW

**Comment by forkata** [comment]
Lots of good stuff :+1:

Do we need to do anything to make sure the buttons continue to work?

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '50a7bd86f75fd1dbacc4dcd3fe547ef1ca8ea93c95afd5f46f9c2117df2f4175ad9ad38e4d05b4b05844261b0a4a523b33cb30f08b4579c63362e4a941c1610e'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (adammathys) [inline config/initializers/devise.rb]: We should change this.

File: db/migrate/20160602043328_drop_users.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class DropUsers < ActiveRecord::Migration[5.0]
+  def up
+    drop_table "users"
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration, "Yeah no that's not going to work."
```
Comment (adammathys) [inline db/migrate/20160602043328_drop_users.rb:7]: :heart:

**Comment by jarednorman** [comment]
Get wrekt. I'm merging this.

**Pull request merged**